<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyBug</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tap">
      <property name="font">
       <font>
        <family>Consolas</family>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="DEBUG">
       <attribute name="title">
        <string>TCP~UART</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="10,1,3">
          <item>
           <widget class="QTextBrowser" name="TB_TCPshow">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="LE_TCP"/>
            </item>
            <item>
             <widget class="QPushButton" name="PB_TCPopen">
              <property name="text">
               <string>Start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PB_ClearTCPshow">
              <property name="text">
               <string>clear</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PB_ClearTCPinput">
              <property name="text">
               <string>clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QTextEdit" name="TE_TCPedit"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="PB_TCPsend">
                <property name="text">
                 <string>Send</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="LAB_TCPrecv">
                <property name="text">
                 <string>RX:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="LAB_TCPsend">
                <property name="text">
                 <string>TX:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="10,1,3">
          <item>
           <widget class="QTextBrowser" name="TB_Serialshow"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="LE_Serialbaud"/>
            </item>
            <item>
             <widget class="QComboBox" name="CB_Serial"/>
            </item>
            <item>
             <widget class="QPushButton" name="PB_Serialopen">
              <property name="text">
               <string>start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PB_ClearSerialinput">
              <property name="text">
               <string>clear</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PB_ClearSerialshow">
              <property name="text">
               <string>clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QTextEdit" name="TE_Serialedit"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="PB_Seralsend">
                <property name="text">
                 <string>Send</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="LAB_Serialrecv">
                <property name="text">
                 <string>RX:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="LAB_Serialsend">
                <property name="text">
                 <string>TX:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PLOT">
       <attribute name="title">
        <string>PLOT</string>
       </attribute>
       <widget class="QCustomPlot" name="plot" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>801</width>
          <height>591</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>570</y>
          <width>222</width>
          <height>32</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="PB_Save">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_Clear">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="CLOUD">
       <attribute name="title">
        <string>CLOUD</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
